packages/cli/src/utils/changelogUtils.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { request } from 'gaxios';
+
+export interface ReleaseInfo {
+  name: string;
+  tag_name: string;
+  body: string;
+  published_at: string;
+  html_url: string;
+}
+
+export async function fetchLatestRelease(): Promise<ReleaseInfo | null> {
```

Comment:
This just fetches the latest release, but you might not be running the latest version. So this should try to find the changelog for the version the user is currently using.